src/process_model_output.R

```diff
@@ -40,7 +40,8 @@ var_data_list <- lapply(vars, function(var) {
 var_data_all <- bind_rows(var_data_list)
 total_storage_data <- var_data_all %>%
   group_by(hruid) %>%
-  summarize(total_storage_today = sum(var_values))
+  summarize(total_storage_today = sum(var_values)) 
+total_storage_data$DOY <- as.numeric(format(as.Date(today), "%j"))
```

Comment:
these were all the same DOY before the group_by above, so I feel safe adding it back here in this manner

Reply:
You can also group by DOY so that that column gets carried through to the summarized output:
`group_by(hruid, DOY)`.